README.md

```diff
@@ -21,8 +21,8 @@ XCP-ng versions.
 They require to install some additional packages first:
 
 ```
-sudo yum install -y genisoimage syslinux grub-tools createrepo_c ltfp
-sudo yum install -y --enablerepo=epel gnupg1 libfaketime
+sudo yum install -y genisoimage syslinux grub-tools createrepo_c
+sudo yum install -y --enablerepo=epel gnupg1 libfaketime ltfp
```

Comment:
What's this new software, `ltfp`? :)

Reply:
BTW, libfaketime is now in XCP-ng repos.

Reply:
Not that new indeed :D 
It's the tool I used for the partial repo-mirroring script.

Reply:
I know `lftp`, but not `ltfp`.

Reply:
Another comment: `lftp` is only required by the optional mirror-repos.sh script, so I would put it in a separate list. The reason we imported libfaketime is to make the ISO generation doable just with packages from the XCP-ng repos. Once `gnupg1` imported, we can reach this goal.